lib/bot.js

```diff
@@ -42,14 +42,14 @@ function getBaseUrl(prjName, config) {
   return configUrl;
 }
 
-
-function generateMsg(configUrl, issue) {
-  return ':jira: <' + configUrl + issue.key + '|*' + issue.key + '*>';
+function generateMsg(config, prjName, issue) {
```

Comment:
i was attempting to standardize the functions so that their inputs (mainly including the config) was common, and building the data we needed from that.